pkg/activator/net/throttler_test.go

```diff
@@ -463,7 +489,7 @@ func TestMultipleActivators(t *testing.T) {
 	}
 
 	// Test with 2 activators, 3 endpoints we can send 1 request and the second times out.
-	tryContext, cancel2 := context.WithTimeout(context.TODO(), 100*time.Millisecond)
+	tryContext, cancel2 := context.WithTimeout(context.TODO(), 20*time.Millisecond)
```

Comment:
Isn't the narrow timeout going to screw us over in prow again?

Reply:
I am going to try to rewrite the whole set of tests differently later. So they _might_ for a while (i.e. a few days).

Reply:
Keep it at 100ms then? Does it harm?

Reply:
The second request will succeed :-(
I'll fix this with event driven stuff, soon.